src/subapps/admin/containers/ElasticSearchQuery.tsx

```diff
@@ -73,29 +61,28 @@ const ElasticSearchQueryContainer: React.FunctionComponent<{
     }));
   };
 
-  const onQueryChange = (query: object) => {
-    setQuery(query);
-  };
+  const onQueryChange = (query: object) => setQuery(query);
 
-  const onChangePageSize = (size: number) => {
+  const onChangePageSize = (size: number) =>
     setPagination({
       size,
       from: 0,
     });
-  };
 
   return (
-    <ElasticSearchQueryForm
-      query={query}
-      response={response}
-      busy={busy}
-      error={error}
-      from={from}
-      size={size}
-      onPaginationChange={onPaginationChange}
-      onQueryChange={onQueryChange}
-      onChangePageSize={onChangePageSize}
-    />
+    <>
```

Comment:
Why is a Fragment needed here? It's just one child element?

Reply:
Missed it, thanks